EasyPost.NetFramework/ScanFormCollection.cs

```diff
@@ -17,7 +17,7 @@ public class ScanFormList : Resource
         ///     Get the next page of scan forms based on the original parameters passed to ScanForm.All().
         /// </summary>
         /// <returns>A new EasyPost.ScanFormList instance.</returns>
```

Comment:
update the comment

Reply:
Resolved in [73a3961](https://github.com/EasyPost/easypost-csharp/pull/223/commits/73a39618c0e40963cd3dd1b750ef53a13e56f3d1)